jwst/dark_current/dark_current_step.py

```diff
@@ -53,6 +55,22 @@ def process(self, input):
             else:
                 dark_model = datamodels.DarkModel(self.dark_name)
 
+            # Store user-defined average_dark_current in model, if provided
+            # A user-defined value will take precedence over any value present
+            # in dark reference file
+            if self.average_dark_current is not None:
```

Comment:
I might suggest splitting this if...else tree (lines 61-72) into its own function to enhance readability of the top-level `process` function.  Then you could more easily write a unit test covering lines 68-69 and 72 to make the codecov bot happy.

Reply:
✅